tests/fixtures/detections/valid_analysis/policies/example_policy_generated_functions.py

```diff
@@ -0,0 +1,33 @@
+IGNORED_USERS = {}
+
+def policy(resource):
+    if resource['UserName'] in IGNORED_USERS:
+        return False
+
+    cred_report = resource.get('CredentialReport', {})
+    if not cred_report:
+        return True
+
+    return cred_report.get('PasswordEnabled', False) and cred_report.get(
+        'MfaActive', False)
+
+def title(resource):
+    return 'THIS IS AN EXAMPLE TITLE'
+
+def alert_context(resource):
+    return {'ip': '1.1.1.1'}
+
+def description(resource):
+    return 'THIS IS AN EXAMPLE DESCRIPTION.'
+
+def destinations(resource):
+    return ["ExampleDestinationName"]
+
+def runbook(resource):
+    return 'THIS IS AN EXAMPLE RUNBOOK VALUE.'
+
+def reference(resource):
+    return 'THIS IS AN EXAMPLE REFERENCE.'
+
+def severity(resource):
+    return 'CRITICAL'
```

Comment:
`severity` should be case insensitive and that logic should be handled behind the scenes. It'd be good to make sure that's working as intended.

```suggestion
    return 'CrItIcAl'
```